zivid_samples/src/sample_capture_assistant.cpp

```diff
@@ -0,0 +1,74 @@
+#include <zivid_camera/Capture.h>
+#include <zivid_camera/CaptureAssistantSuggestSettings.h>
+#include <sensor_msgs/Image.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <ros/ros.h>
+
+#define CHECK(cmd)                                                                                                     \
+  do                                                                                                                   \
+  {                                                                                                                    \
+    if (!cmd)                                                                                                          \
+    {                                                                                                                  \
+      throw std::runtime_error{ "\"" #cmd "\" failed!" };                                                              \
+    }                                                                                                                  \
+  } while (false)
+
+namespace
+{
+const ros::Duration default_wait_duration{ 30 };
+constexpr auto ca_suggest_settings_service_name = "/zivid_camera/capture_assistant/suggest_settings";
```

Comment:
You set this duration and service name parameter in both the C++ and the Python sample. Perhaps you could fetch these parameters from the parameter server, using the launch file?

This way you could avoid syncing the parameters in all three places, and also avoid recompiling if you change the parameters.

Reply:
I think we want the samples simple and standalone, so the service name should be visible in the samples. Having too much code/constants that is shared and passed along makes the samples less clear. For example, user may just duplicate our sample cpp or py and get started. They probably dont want to use our sample.launch file.

Reply:
I agree that simplicity of the samples can be worth the trade-off of having to update these constants several places in case they change.